torchtune/models/qwen2_5/_model_builders.py

```diff
@@ -0,0 +1,629 @@
+        embed_dim=3584,
+        intermediate_dim=18944,
+        max_seq_len=32768,
+        attn_dropout=0.0,
+        norm_eps=1e-06,
+        rope_base=1000000.0,
+    )
+
+
+def qwen2_0_5b() -> TransformerDecoder:
+    """
+    Builder for creating a Qwen2.5 model initialized w/ the default 0.5B parameter values
+    from https://huggingface.co/Qwen/Qwen2.5-0.5B-Instruct
+
+    Returns:
+        TransformerDecoder: Instantiation of Qwen2.5 0.5B model
+
+    Note:
+        Qwen2.5 0.5B-3B model builders will enable `tie_word_embeddings` by default
+        and returns an instance of `TransformerDecoder`.
+    """
+    return qwen2(
+        vocab_size=151936,
+        num_layers=24,
+        num_heads=14,
+        num_kv_heads=2,
+        embed_dim=896,
+        intermediate_dim=4864,
+        max_seq_len=32768,
+        attn_dropout=0.0,
+        norm_eps=1e-06,
+        rope_base=1000000.0,
+        tie_word_embeddings=True,
+    )
+
+
+def qwen2_1_5b() -> TransformerDecoder:
+    """
+    Builder for creating a Qwen2.5 model initialized w/ the default 1.5B parameter values
+    from https://huggingface.co/Qwen/Qwen2.5-1.5B-Instruct
+
+    Returns:
+        TransformerDecoder: Instantiation of Qwen2.5 1.5B model
+
+    Note:
+        Qwen2.5 0.5B-3B model builders will enable `tie_word_embeddings` by default
+        and returns an instance of `TransformerDecoder`.
+    """
+    return qwen2(
+        vocab_size=151936,
+        num_layers=28,
+        num_heads=12,
+        num_kv_heads=2,
+        embed_dim=1536,
+        intermediate_dim=8960,
+        max_seq_len=32768,
+        attn_dropout=0.0,
+        norm_eps=1e-06,
+        rope_base=1000000.0,
+        tie_word_embeddings=True,
+    )
+
+
+def qwen2_3b() -> TransformerDecoder:
+    """
+    Builder for creating a Qwen2.5 model initialized w/ the default 3B parameter values
+    from https://huggingface.co/Qwen/Qwen2.5-3B-Instruct
+
+    Returns:
+        TransformerDecoder: Instantiation of Qwen2.5 3B model
+
+    Note:
+        Qwen2.5 0.5B-3B model builders will enable `tie_word_embeddings` by default
+        and returns an instance of `TransformerDecoder`.
+    """
+    return qwen2(
+        vocab_size=151936,
+        num_layers=36,
+        num_heads=16,
+        num_kv_heads=2,
+        embed_dim=2048,
+        intermediate_dim=11008,
+        max_seq_len=32768,
+        attn_dropout=0.0,
+        norm_eps=1e-06,
+        rope_base=1000000.0,
+        tie_word_embeddings=True,
+    )
+
+
+def qwen2_14b() -> TransformerDecoder:
+    """
+    Builder for creating a Qwen2.5 model initialized w/ the default 14B parameter values
+    from https://huggingface.co/Qwen/Qwen2.5-14B-Instruct
+
+    Returns:
+        TransformerDecoder: Instantiation of Qwen2.5 14B model
+    """
+    return qwen2(
+        vocab_size=152064,
+        num_layers=48,
+        num_heads=40,
+        num_kv_heads=8,
+        embed_dim=5120,
+        intermediate_dim=13824,
+        max_seq_len=32768,
```

Comment:
The config [here](https://huggingface.co/Qwen/Qwen2.5-14B-Instruct/blob/main/config.json) has `"max_position_embeddings": 32768`. 

It has `"sliding_window": 131072` but I don't think this matters since `"use_sliding_window": false`